code/weapon/trails.cpp

```diff
@@ -304,7 +304,9 @@ void trail_render( trail * trailp )
 	g3_render_primitives_colored_textured(&material_def, Trail_v_list, nv, PRIM_TYPE_TRISTRIP, false);
 }
 
-void trail_add_segment( trail *trailp, vec3d *pos )
+// Adds a new segment to trailp at pos, in order to do spread an orient is provided 
+// so it can drift away orthogonally from the direction of the trail
+void trail_add_segment( trail *trailp, vec3d *pos , const matrix* orient)
```

Comment:
```suggestion
// Adds a new segment to trailp at pos.
// In order for trailp's `spread` field to have any effect, it must be nonzero and the orient must be non-null.
// The orient is relative to [what?]
// [It can drift away orthogonally from the direction of the trail] <-- what is "it"?
void trail_add_segment( trail *trailp, vec3d *pos , const matrix* orient)
```

Reply:
Ok, I think I see what you mean.